docs/contents/ions.rst

```diff
@@ -0,0 +1,109 @@
+Ions
+====
+
+NEURON supports computing the ion currents. For each segment there can be
+separate current for separate ions, i.e. one for sodium ions and another for
+calcium ions.
+
+There are five variables associated with ions: the current (``ina``), the
+concentration inside the segment (``nai``), the concentration outside the
```

Comment:
... inside the segment adjacent to the membrane

... outside the segment adjacent to the membrane

(the reason I mention it is that many ion accumulation models discretize the radial dimension to model radial diffusion.

Reply:
Added, thank you for the useful explanation.